sklearn/tree/_classes.py

```diff
@@ -686,9 +686,76 @@ def feature_importances_(self):
             Normalized total reduction of criteria by feature
             (Gini importance).
         """
+        return self._unnormalized_feature_importances / np.sum(
+            self._unnormalized_feature_importances
+        )
+
+    @property
+    def _unnormalized_feature_importances(self):
+        check_is_fitted(self)
+
+        return self.tree_.compute_feature_importances(normalize=False)
+
+    def compute_unbiased_feature_importance(self, X_test, y_test, sample_weight=None):
```

Comment:
In line with the long method name above, should we add a `return_oob_pred` option here ?